### PR TITLE
Add the script to build h5py and shapely 'manylinux' wheels

### PR DESCRIPTION
**Pull request by daniviga**
The build is made on the pypi official CentOS 5 docker container to be compliant with [PEP 513](https://www.python.org/dev/peps/pep-0513/#id42).
Wheels are made for cp27m, cp27mu, cp34m, cp35m

Docker is required.

### CODE REVIEW

**Comment by micheles** [comment]
Do we need to support 3.4 on some platform or cp34m is a typo in the description above?

**Comment by daniviga** [comment]
It is not a typo. It's up to you, it does cost nothing supporting it, 5 mins in building the wheels. I excluded cp33m because even numpy does not have wheels for it.

**Comment by micheles** [comment]
Building the wheels is easy, but then we need to enable python 3.4 in travis, double the time of the tests and most importantly, if something breaks I have to fix it. So I would stick with Python 3.5 until there is a good reason (i.e. a paying sponsor) to support it.

**Comment by daniviga** [comment]
np, btw just to be clear, here I'm building just a couple of generic wheels, nothing strictly related to OQ.

File: unix/wheels/build-wheels-docker.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+rm -Rf $OQ_PREFIX
+mkdir -p $OQ_PREFIX/src
+
+cd $OQ_PREFIX/src
+
+# Get sources
+curl -Lo libgeos-3.5.0.tar.gz https://github.com/libgeos/libgeos/archive/3.5.0.tar.gz
+curl -LO http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.17.tar.gz
+curl -LO https://pypi.python.org/packages/22/82/64dada5382a60471f85f16eb7d01cc1a9620aea855cd665609adf6fdbb0d/h5py-2.6.0.tar.gz
+curl -LO https://pypi.python.org/packages/ad/f9/4640d50324635fbdc7b109f8ef37de5f04456b89ed175cf2f71ae05efd8f/Shapely-1.5.13.tar.gz
```
Comment (nastasi-oq) [inline unix/wheels/build-wheels-docker.sh]: No other way to retrieve these files ?
For example:
curl -o pageout.html 'https://pypi.python.org/pypi/h5py/2.6.0'
more some xmlstarlet could be a better option to get the tgz url.

**Comment by daniviga** [comment]
Why? That looks quite over engineered. This scripts does not run every day, is only needed if we loose the whl files or if we need to build a new version and in that case we need to upgrade the script anyway.

File: unix/wheels/build-wheels-docker.sh (hunk 1 above)
Comment (daniviga) [inline unix/wheels/build-wheels-docker.sh]: Why? That looks quite over engineered. This scripts does not run every day, is only needed if we loose the whl files or if we need to build a new version and in that case we need to upgrade the script anyway.

**Comment by daniviga** [comment]
~~Must fix https://github.com/gem/oq-installers/issues/24 first~~

**Comment by nastasi-oq** [comment]
LGTM

**Pull request merged**